static/js/webrtc.js

```diff
@@ -182,8 +184,24 @@ var rtc = (function() {
     toggleVideo: function() {
       var videoTrack = localStream.getVideoTracks()[0];
       if (videoTrack) {
-        videoTrack.enabled = !videoTrack.enabled;
-        return !videoTrack.enabled;
+        if (videoTrack.enabled) {
+          videoTrack.enabled = false;
+          videoTrack.stop()
+          return false
+        } else {
+          var audioTrack = localStream.getAudioTracks()[0];
+          console.log(1, audioTrack)
+          var audioTrackEnabled = audioTrack.enabled;
+          self.deactivate()
+          self.activate()
```

Comment:
I've been experimenting; this is the only way I can figure out so far, how to recover from `.stop()`. I can always try a stripped down version of them as well. Right now I'm experimenting with just caling `getUserMedia` and adding the video to the existing stream. The whole hierarchy of things is a bit confusing so I don't know if that's appropriate though.